docs/source/docs/hardware/selecting-hardware.md

```diff
@@ -11,27 +11,19 @@ In order to use PhotonVision, you need a coprocessor and a camera. This page wil
 - CPU: ARM Cortex-A53 (the CPU on Raspberry Pi 3) or better
 - At least 8GB of storage
 - 2GB of RAM
-    - PhotonVision isn't very RAM intensive, but you'll need at least 2GB to run the OS and PhotonVision.
+  - PhotonVision isn't very RAM intensive, but you'll need at least 2GB to run the OS and PhotonVision.
 - The following IO:
-    - At least 1 USB or MIPI-CSI port for the camera
-        - Note that we only support using the Raspberry Pi's MIPI-CSI port, other MIPI-CSI ports from other coprocessors may not work.
-    - Ethernet port for networking
+  - At least 1 USB or MIPI-CSI port for the camera
+    - Note that we only support using the Raspberry Pi's MIPI-CSI port, other MIPI-CSI ports from other coprocessors will not work.
```

Comment:
This is dependent on how V4 L is feeling, but I'm not aware of any coprocessor that it cooperates on

Reply:
True I guess.